cinema/views.py

```diff
@@ -87,6 +111,8 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
         )
     )
     serializer_class = MovieSessionSerializer
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
+    http_method_names = ['get', 'post', 'put', 'patch', 'delete'] 
```

Comment:
It's not a good practice to allow all types of HTTP methods on your endpoints. It's better to be explicit about which HTTP methods are allowed. For example, if you don't want to allow DELETE requests, you should remove 'delete' from the list of `http_method_names`. This way, you can prevent accidental data loss.